v2/device.go

```diff
@@ -145,6 +145,12 @@ func (s *DeviceClient) Publish(topic string, qos byte, retained bool, payload in
 	return &mqtt.DummyToken{}
 }
 
+// PublishSimple publishes a message without queueing to the internal queue.
+// The messages will be lost when they are published offline.
```

Comment:
might be better to comment that this is not a function specified in mqtt.Client interface.